Launcher/Classes/LanguageController.swift

```diff
@@ -0,0 +1,37 @@
+//
+//  LanguageController.swift
+//  Calabash Launcher
+//
+//  Created by Serghei Moret on 16.10.17.
+//  Copyright © 2017 XING. All rights reserved.
+//
+
+import Cocoa
+import Foundation
+import AppKit
+import CommandsCore
+
+class LanguageController: NSViewController {
```

Comment:
I would name it ‘LanguageViewController’ if it’s a view controller

Reply:
... and then also rename the file + header. :)